### PR TITLE
Created compact agenda view page

### PR DESCRIPTION
**Pull request by FrancescaFr**
Created separate compact agenda view page and corresponding agenda card component + "compact" CSS.
Added button link on original agenda page.
Deviates from aesthetic of other pages a bit and does not include Hero component to improve page print-ability.
(noted limitation: changes in schedule made to original Agenda page will not be reflected here automatically)

### CODE REVIEW

**Comment by TheJoin95** [comment]
Overall it's looking good. Could you please integrate the changes from #500 into the new agenda page?

File: pages/agenda.tsx [hunk 1]
```diff
@@ -51,6 +52,9 @@ export default function Agenda() {
           <div className='agenda-nav'>
             {NavBar(1)}
             {NavBar(2)}
+            <div className="compact button">
+              <Link href={'/agenda-compact'}>Compact View</Link>
+            </div>
```
Comment (TheJoin95) [inline pages/agenda.tsx]: Let's add the button as the first element of the agenda nav with a normal size button.

File: styles/globals.css [hunk 2]
```diff
@@ -1264,6 +1264,51 @@ div.image .img {
   margin-bottom: 10em;
 }
 
+.compact {
```
Comment (TheJoin95) [inline styles/globals.css:1271]: Could you please add a `media print` css query to exclude the webiste navbar, footer and other useless elements while printing the schedule?

**Review by TheJoin95 — CHANGES_REQUESTED** [review]
Few remarks

File: pages/agenda-compact.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    </span>
+  )
+  
+  return (
+    <>
+          <div className=".container">
+
+        <div className='agenda-nav-container'>
+          <div className='agenda-nav compact'>
+            {ViewButton}
```
Comment (TheJoin95) [inline pages/agenda-compact.tsx]:
```suggestion
          {ViewButton}
          <div className='agenda-nav compact'>
```

File: pages/agenda-compact.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+            Click to follow the live stream on YouTube!
+          </a>
+        </div>}
+
+        <div className="container">
+
+          {trackSelected === 'Alpha1' && (
+            <>
+              <div className={`agenda-nav compact`}>
+                <h1 > OS Day 1 - Alpha Track (07.04)</h1>
```
Comment (Wabri) [inline pages/agenda-compact.tsx]:
```suggestion
                <h1 > OS Day 1 - Alpha Track (07.03)</h1>
```

File: pages/agenda-compact.tsx [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+              <AgendaCardCompact hour='16:40' isBreak={false} speakerName={s('sal.name')} talkTitle={s('sal.talk_title')} icon='/icons/microphone.svg'   link={s('sal.id')}/>
+              <AgendaCardCompact hour='17:20' isBreak={false} speakerName={s('francesco.name')} talkTitle={s('francesco.talk_title')} icon='/icons/microphone.svg'   link={s('francesco.id')}/>
+              <AgendaCardCompact hour='18:30' isBreak={true} breakTitle="Aperitivo" icon='/icons/drink.svg'  />
+            </>
+          )}
+
+          {trackSelected === 'Beta1' && (
+            <>
+              <div className={`agenda-nav compact`}>
+                <h1> OS Day 1 - Beta Track (07.04)</h1>
```
Comment (Wabri) [inline pages/agenda-compact.tsx]:
```suggestion
                <h1> OS Day 1 - Beta Track (07.03)</h1>
```

File: pages/agenda-compact.tsx [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+              <AgendaCardCompact hour='16:40' isBreak={false} speakerName={s('noah.name')} talkTitle={s('noah.talk_title')} icon='/icons/microphone.svg'   link={s('noah.id')}/>
+              <AgendaCardCompact hour='17:20' isBreak={false} speakerName={s('francesco.name')} talkTitle={s('francesco.talk_title')} icon='/icons/microphone.svg'   link={s('francesco.id')}/>
+              <AgendaCardCompact hour='18:30' isBreak={true} breakTitle="Aperitivo" icon='/icons/drink.svg'  />
+            </>
+          )}
+
+          {trackSelected === 'Alpha2' && (
+            <>
+              <div className={`agenda-nav compact`}>
+                <h1> OS Day 2 - Alpha Track (08.04)</h1>
```
Comment (Wabri) [inline pages/agenda-compact.tsx]:
```suggestion
                <h1> OS Day 2 - Alpha Track (08.03)</h1>
```

File: pages/agenda-compact.tsx [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+              {/* <AgendaCardCompact hour='16:00' isBreak={true} breakTitle='Coffee break & Giveaway' icon='/icons/coffee.svg'  /> */}
+              <AgendaCardCompact hour='17:00' isBreak={false} speakerName={s('andrey.name')} talkTitle={s('andrey.talk_title')} icon='/icons/microphone.svg'   link={s('andrey.id')}/>
+              <AgendaCardCompact hour='18:30' isBreak={true} breakTitle="Schrödinger Hat Final greetings" icon='/icons/microphone.svg'  />
+            </>
+          )}
+
+          {trackSelected === 'Beta2' && (
+            <>
+              <div className={`agenda-nav compact`}>
+                <h1> OS Day 2 - Beta Track (08.04)</h1>
```
Comment (Wabri) [inline pages/agenda-compact.tsx]:
```suggestion
                <h1> OS Day 2 - Beta Track (08.03)</h1>
```

**Review by Wabri — DISMISSED** [review]
Beautiful work! Thanks 💯 

Few minor fixes and we are good to go!

File: pages/agenda-compact.tsx [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+          Day{day} - {track} Track
+        </span>;
+    });
+  };
+
+  const ViewButton = (
+     <span 
+      className='nav-button'>
+      <Link href={'/agenda'}>Timeline View</Link>
+    </span>
```
Comment (TheJoin95) [inline pages/agenda-compact.tsx]:
```suggestion
     <div 
      className='nav-button'>
      <Link className="button" href={"/agenda"}>Timeline View</Link>
    </div>
```

File: styles/globals.css [hunk 9]
```diff
@@ -1264,6 +1264,51 @@ div.image .img {
...
+  padding: .1em;
+  margin-top: -0.25em;
+  margin-left: 1em;
+  margin-right:1em;
+}
+
```
Comment (TheJoin95) [inline styles/globals.css:1277]: Add the following:
```
.nav-button {
    padding: .5em;
}
```

So that the button can align itself with the agenda navbar

Comment (TheJoin95) [reply styles/globals.css:1277]: This should be the final design:
image

File: pages/agenda-compact.tsx [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+  const ViewButton = (
+     <div 
+      className='nav-button'>
+      <Link className="button" href={"/agenda"}>Timeline View</Link>
+    </div>
+  )
+  
+  return (
+    <>
+          <div className=".container">
```
Comment (TheJoin95) [inline pages/agenda-compact.tsx]:
```suggestion
          <div className="container">
```

File: pages/agenda.tsx [hunk 11]
```diff
@@ -33,6 +34,13 @@ export default function Agenda() {
     });
   };
 
+  const ViewButton = (
+    <span 
+     className='nav-button'>
+     <Link href={'/agenda-compact'}>Compact View</Link>
+   </span>
```
Comment (TheJoin95) [inline pages/agenda.tsx]:
```suggestion
    <div 
      className="nav-button">
      <Link className="button" href={"/agenda-compact"}>Compact View</Link>
    </div>
```

**Review by TheJoin95 — APPROVED** [review]
Thanks for the effort and sorry for the long / on spot review. Looking good

**Comment by TheJoin95** [comment]
@Wabri could you please have a quick look? So I can merge it

**Pull request merged**